Assets/Scripts/Layers/LayerTypes/HierarchicalObjectLayerGameObject.cs

```diff
@@ -68,62 +75,72 @@ protected void OnDestroy()
 
         private void UpdatePosition(Coordinate newPosition)
         {
-            if (newPosition.ToUnity() != transform.position)
-                transform.position = newPosition.ToUnity();
+            if (newPosition.ToUnity() == worldTransform.Coordinate.ToUnity()) return;
+
+            worldTransform.Coordinate = newPosition;
+            transform.position = worldTransform.Coordinate.ToUnity();
         }
 
         private void UpdateRotation(Vector3 newAngles)
         {
-            if (newAngles != transform.eulerAngles)
-                transform.eulerAngles = newAngles;
+            if (newAngles == transform.eulerAngles) return;
+
+            worldTransform.Rotation = Quaternion.Euler(newAngles);
+            transform.rotation = worldTransform.Rotation;
```

Comment:
gaat dit goed met de bolling van de aarde? moet er niet ook een ToUnity() functie worden aangeroepen?

Reply:
En anders draai ik de rotatie dingen even terug; voor de zekerheid

Reply:
worldtransform.rotation is de rotatie van het gameobject in zijn eigen coordinatenstelsel. (dus vóór het toepassen van de "bolling", zoals mike het noemt)
de rotatie in unity wordt uitgerekend met worldtransform.coordinate.RotationToUnityUp() * worldtransform.rotation